docs/source/collect/certificates.md

```diff
@@ -27,16 +38,16 @@ metadata:
 spec:
   collectors:
     - certificates: 
-      secrets:
-        - name: envoycert
-          namespaces:
-            - kube-system
-            - projectcontour
-      configMaps:
-        - name: kube-root-ca.crt
-          namespaces:
-            - curlie
-            - kurl
+        secrets:
+          - name: envoycert
+            namespaces:
+              - kube-system
+              - projectcontour
```

Comment:
```suggestion
          - name: envoycert
             namespaces:
               - kube-system
               - projectcontour
           - name: kube-root-ca.crt
              namespaces:
                - default
                - kube-public
```

Reply:
Indicate by example that this is indeed a list of objects

Reply:
list of objects added